### DISCUSSION TITLE
service_docker_stdout采集不到容器日志

### DISCUSSION THREAD

**Discussion by gagraler** [alibaba/ilogtail#1428] [Help] [answered]
cm配置如下：
![906a2302ed43419355d0bbdc727ee10](https://github.com/alibaba/ilogtail/assets/91018856/5cd9fb12-5f99-4ecc-b0bb-92225294fa6e)

ilogtail pod日志输出如下：
![cfe7e2c17f3b6439a01d9e6b9e84bf3](https://github.com/alibaba/ilogtail/assets/91018856/7c42a93a-fee9-48df-ae15-9b4f5388fd43)

需要采集的容器详情：
![image](https://github.com/alibaba/ilogtail/assets/91018856/be6bc624-2403-4253-b929-221944bef02f)
![image](https://github.com/alibaba/ilogtail/assets/91018856/95828b21-cf1a-42ed-b69f-45e4cf76b024)

容器日志输出如下：
![image](https://github.com/alibaba/ilogtail/assets/91018856/a24ade94-7308-42b0-9e9d-e261d4603994)

现需求是，采集容器输出到es or kafak，我这里用的es

**Comment by onejimmyboy** [comment, marked as answer]
这日志输出不是ilogtail的日志吧，还有采集配置相关的先去掉多行匹配试试，排除是不是多行匹配正则的问题。
